components/drops/useDrops.ts

```diff
@@ -181,6 +183,28 @@ export function useDrop(alias?: string) {
 
   const fetchDrop = async () => {
     drop.value = await getDropById(alias ?? params.id.toString())
+
+    if (drop.value.collection) {
+      if (isSub.value) {
+        const { maxSupply: supply, minted, metadata } = await subCollection(drop.value.collection)
+
+        drop.value.max = supply
+        drop.value.minted = minted
+        drop.value.collectionName = metadata.name
+        drop.value.collectionDescription = metadata.description
+      }
+
+      if (isEvm.value) {
+        const { urlPrefix } = usePrefix()
+        const address = drop.value.collection as `0x${string}`
+        const { maxSupply: supply, metadata, minted } = await evmCollection(address, urlPrefix.value)
+
+        drop.value.max = supply
+        drop.value.minted = minted
+        drop.value.collectionName = metadata.name
+        drop.value.collectionDescription = metadata.description
+      }
+    }
```

Comment:
I would avoid nesting ifs were posible

```suggestion
    if (!drop.value.collection) {
       return
    }
    
     if (isSub.value) {
      const { maxSupply: supply, minted, metadata } = await subCollection(drop.value.collection)

      drop.value.max = supply
      drop.value.minted = minted
      drop.value.collectionName = metadata.name
      drop.value.collectionDescription = metadata.description
    }

    if (isEvm.value) {
      const { urlPrefix } = usePrefix()
      const address = drop.value.collection as `0x${string}`
      const { maxSupply: supply, metadata, minted } = await evmCollection(address, urlPrefix.value)

      drop.value.max = supply
      drop.value.minted = minted
      drop.value.collectionName = metadata.name
      drop.value.collectionDescription = metadata.description
    }
```

Reply:
oh, agree, that's a better one. let me update